### PR TITLE
Understanding "same-site" and "same-origin"

### LINKED ISSUE #2500: content: 'same-site' & 'same-origin' [apr 13]
[draft](https://docs.google.com/document/d/1bprwXaQM6ZD5K8iEMnx-n-iKpNeCTSp3pabRor2YUVI/edit#)

deadline: apr 13 (hard)

### PR DESCRIPTION
**Pull request by agektmr**
Fixes #2500

### CODE REVIEW

**Comment by kaycebasques** [comment]
@agektmr you've got a merge conflict

**Comment by agektmr** [comment]
@kaycebasques the conflict should have been resolved.

File: src/site/content/en/secure/same-site-same-origin/index.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+date: 2020-04-15
+description: |
+  "same-site" and "same-origin" are frequently cited but often misunderstood
+  terms. This article helps you understand what they are and how they are
+  different.
+---
+
+"same-site" and "same-origin" are frequently cited but often misunderstood
+terms. For example, they are mentioned in the context of page transitions,
+`fetch()` requests, cookies, opening popups, and embedding resources and
```
Comment (kaycebasques) [inline src/site/content/en/secure/same-site-same-origin/index.md:15]:
```suggestion
`fetch()` requests, cookies, opening popups, embedded resources, and
```

File: src/site/content/en/secure/same-site-same-origin/index.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+the combination of the TLD and the part of the domain just before it. For
+example, given a URL of `https://www.example.com:443/foo` , the "site" is
+`example.com`.
+
+However, for domains such as `.co.jp` or `.github.io`, just using the TLD of
+`.jp` or `.io` is not granular enough to identify the "site". And there is no
+way to algorithmically determine the level of registrable domains for a
+particular TLD. That's why a list of "effective TLDs" was created. These are
+defined in the [Public Suffix List. The list of
+eTLDs](https://wiki.mozilla.org/Public_Suffix_List) is maintained at
```
Comment (kaycebasques) [inline src/site/content/en/secure/same-site-same-origin/index.md:95]:
```suggestion
defined in the [Public Suffix List](https://wiki.mozilla.org/Public_Suffix_List).
 The list of eTLDs is maintained at
```

File: src/site/content/en/secure/same-site-same-origin/index.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+        <td><strong>schemeful same-site: ports don't matter</strong></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## How to check if a request is "same-site", "same-origin", or "cross-site"
+
+Chrome sends requests along with a `Sec-Fetch-Site` HTTP header (No other
+browsers support as of April 2020). This is part of a larger [Fetch Metadata
```
Comment (kaycebasques) [inline src/site/content/en/secure/same-site-same-origin/index.md:203]:
```suggestion
Chrome sends requests along with a `Sec-Fetch-Site` HTTP header. No other
browsers support `Sec-Fetch-Site` as of April 2020. This is part of a larger [Fetch Metadata
```

**Comment by kaycebasques** [comment]
I'm pretty sure the Percy issue is not a problem. It's just showing that the text `6 resources` for the Safe & Secure collection changed to `7 resources` which is to be expected.

**Pull request merged**